firrtl/src/test/scala/firrtlTests/SerializerSpec.scala

```diff
@@ -98,6 +98,30 @@ object SerializerSpec {
       "test"
     )
 
+  val constModule: String =
+    """module test :
+      |
+      |  wire constInt : const UInt<3>
+      |  wire constAsyncReset : const AsyncReset
+      |  wire constBundle : const { real : UInt<32>, imag : UInt<32>, other : const SInt<1>}""".stripMargin
+
+  val constsModuleIR: Module =
+    Module(
+      NoInfo,
+      "test",
+      Seq.empty,
+      Block(
+        Seq(
+          DefWire(NoInfo, "constInt", ConstType(UIntType(IntWidth(3)))),
+          DefWire(NoInfo, "constAsyncReset", ConstType(AsyncResetType)),
+          DefWire(NoInfo, "constBundle", ConstType(BundleType(Seq(
+            Field("real", Default, UIntType(IntWidth(32))),
+            Field("imag", Default, UIntType(IntWidth(32))),
+            Field("other", Default, ConstType(SIntType(IntWidth(1)))),
+          )))),
+        )
+      )
+    )
```

Comment:
Nit: this may be slightly cleaner as simpler tests of each `Serializer.serialize(DefWire(...)) should be "wire ..."` instead of making it a complete module. This may then be short enough to inline all the tests into one-line-per test.